include/gz/utils/ImplPtr.hh

```diff
@@ -25,10 +25,11 @@
 #include <gz/utils/SuppressWarning.hh>
 #include <gz/utils/Export.hh>
 
-namespace ignition
+namespace gz
 {
   namespace utils
   {
+    inline namespace IGNITION_UTILS_VERSION_NAMESPACE {
```

Comment:
Good point, I believe they cause issues on Windows

Reply:
Ah, right, we should use it since we're ticktocking the macros

I'm not sure why this was added, IIRC it was for when the namespace was still on `ignition`. But now that it's on `gz` it's unnecessary, so we can remove it? pinging @chapulina for confirmation.

Reply:
I vaguely remember that too, but I may be confusing with the `_VISIBILITY` macros.

I think we shouldn't be adding any namespaces that aren't there already in this series of PRs.

Reply:
resolved by https://github.com/gazebosim/gz-utils/pull/61/commits/c27b7ac3b3a18718a2e3ee17ef4147ab2fd5826e